test/config/config.yaml

```diff
@@ -31,6 +31,27 @@ peers:
       p2p: 18333
       zmq: 28332
 
+broadcasting:
+  mode: unicast
+  ipv6Enabled: true
+  multicastGroup: 172.28.56.77
```

Comment:
Thinking now maybe it would be better to have a distinct multicast object or unicast object like this 🤔 

Also comments should be added for each new setting

```yaml
broadcasting: # settings for connection to nodes
  mode: unicast # one of unicast | multicast
  multicast: # must be specified if mode = multicast
    ipv6Enabled: true # indicates whether ipv6 is enabled for multicasting
    multicastGroup: 172.28.56.77 # address of multicast group, needs to be ipv6 address if ipv6 is enabled
    interfaces: # network interfaces
      - "eth0"
      - "eth1"
  unicast: # must be specified if mode = unicast
    peers: # list of bitcoin node peers to connect to
    - host: node1
      port:
        p2p: 18333
        zmq: 28332
    - host: node2
      port:
        p2p: 18333
        zmq: 28332
    - host: node3
      port:
        p2p: 18333
        zmq: 28332
```

Reply:
Config is according to the scheme above